res/css/views/right_panel/_ThreadPanel.scss

```diff
@@ -120,6 +120,13 @@ limitations under the License.
     }
 
     &.mx_ThreadView {
+        max-height: 100%;
+
+        .mx_ThreadView_List {
```

Comment:
I cannot find `mx_ThreadView_List` anywhere in the code. Can you double-check and eventually remove this instead of moving it around :smile:

Reply:
Ach thanks for checking!